### PR TITLE
"minimum viable product" shiny interface to LEMMA

### PR DESCRIPTION
**Pull request by slwu89**
This PR makes a new folder `inst/shiny` in the package which contains the ui and server code for the shiny app, and a file `app.R` in the `R` directory which runs the app when calling `LEMMA::run_app()`.

The main splash page of app just displays the contents of the markdown document `inst/shiny/src/SEIRmodel.md`, and can be edited to include more mathematical details, author affiliations, statistical notes, etc. 

The dropdown tabset "Excel Interface" has two panels, "Data Input" lets a user download the template `example.xlsx` file to their computer, and upload a .xlsx configuration file to the shiny app. On the right the markdown document `inst/shiny/src/excel_input.md` is displayed, which currently is just copied from the LEMMA website explaining what each sheet of the config .xlsx should contain. After the file is uploaded, the panel "Run LEMMA" can be selected; after pressing (and waiting) the `Run LEMMA` button, the .xlsx and .pdf outputs can be downloaded with the download buttons. The right hand side just displays the output of the projection data, to make sure it is working properly.

@joshuaschwab I can't tag you as a reviewer for some reason (I might need to be a collaborator on the repo to do that) but consider this a review request = ]

### CODE REVIEW

**Review by joshuaschwab — CHANGES_REQUESTED** [review]
This looks good, except 
`z=CredibilityIntervalFromExcel("test_SW.xlsx")`
should have
```
names(z)
[1] "fit.to.data"  "fit.extended" "projection"   "gplot"        "excel.output" "inputs"      
```
but instead has
```
names(z)  
 [1] "fit.to.data"                         "fit.extended"                        "projection"                         
 [4] "inputs"                              "gplot.short.term"                    "gplot.long.term"                    
 [7] "gplot.rt"                            "excel.output.projection"             "excel.output.posteriorParams"       
[10] "excel.output.posteriorInterventions" "excel.output.all.inputs"  
```

File: R/CredibilityInterval.R [hunk 1]
```diff
@@ -33,10 +33,19 @@ CredibilityInterval <- function(inputs, fit.to.data = NULL) {
   
   excel.output <- GetExcelOutput(ci_data$projection, ci_data$fit.to.data, ci_data$inputs)
   gplot <- GetPdfOutput(ci_data$fit.extended, ci_data$projection, ci_data$inputs)
-  # invisible(list(fit.to.data = fit.to.data, fit.extended = fit.extended, projection = projection, gplot = gplot, excel.output = excel.output, inputs = inputs.copy))
-  invisible(c(
-    ci_data, gplot = gplot, excel.output = excel.output
-  ))
+  invisible(
+    list(
```
Comment (slwu89) [inline R/CredibilityInterval.R:37]: @joshuaschwab thanks for the catch! It should output the expected set of objects now:

```
> names(z)
[1] "fit.to.data"  "fit.extended" "projection"   "gplot"        "excel.output" "inputs"    
```

Comment (joshuaschwab) [reply R/CredibilityInterval.R:37]: Thanks, that looks better, but I see`z$inputs` is NULL

Comment (slwu89) [reply R/CredibilityInterval.R:37]: Ah, it should be working now after the last push! I had to slightly rearrange the internal functions to be more shiny-friendly and got a few things mixed up. It should return the same values as the the main branch on your repo now.

Comment (joshuaschwab) [reply R/CredibilityInterval.R:37]: Thanks!

**Pull request merged**